docs/account_management/xias/index.md

```diff
@@ -11,4 +11,18 @@ An overview of creating an external collaborator account:
     2. [Create and manage XIAS Guests](pi_guest_management.md)
 2. The guest must then [Create a XIAS Guest Account](guest_instructions.md)
 
+## What is next and how to login to services?
+
+To login to the Cheaha and gitlab services, you need to use your XIAS email address and a password  used to create the XIAS account as credentials. For accessing the cloud service, your username is your XIAS email address, while the password can be any current password you have, not necessarily the same as the one used to create your XIAS email address. For further guidance on how to proceed and what steps to take next, please refer to the following documentation pages.
```

Comment:
- There is a double space between in `and a password  used to create`.
- I think it should be "the password used to create your XIAS account".
- "GitLab" should be capitalized like so.
- Replace "the cloud service" with "Cloud.rc" and crosslink to https://docs.rc.uab.edu/uab_cloud/
- Also crosslink "Cheaha" to https://docs.rc.uab.edu/cheaha/getting_started/
- Also crosslink "GitLab" to https://docs.rc.uab.edu/account_management/gitlab_account/

To access Cloud.rc, they first need to request an account. It may be best to avoid describing how to access it here, and instead link them directly to https://docs.rc.uab.edu/uab_cloud/#first-steps.